### DISCUSSION TITLE
ChromeBrew logo change for my local fork.

### DISCUSSION THREAD

**Discussion by Alex313031** [skycocker/chromebrew#5009] [Show and tell]
I know this is GNU, but I wanted to ask out of niceties and to maybe get a source/high res image, of the logo.
I simply want to modify the chrome logo into the blue one, to walk in hand with my repo description change. I would give credit and info about the logo change in the main md as well as link to this official repo, to ensure anyone coming across my fork knows I am merely a peasant in my scripting knowledge who bows to the people who's hard work make stuff like this awesome package manager possible.

Also side note (will also file a bug about this): Removing crew via the wiki [https://github.com/skycocker/chromebrew/wiki/How-to-remove-chromebrew](url) leaves a simlink issue that makes reinstalling crew impossible. Unlinking and rm -rf or creating then removing /usr/local/var doesn't solve anything. The error given is "unlink: cannot unlink '/usr/local/var': Is a directory"

**Comment by supechicken** [comment]
Do you mean something like 'Chromiumbrew' ?
